packages/create-react-native-library/templates/common/$package.json

```
@@ -70,7 +70,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "devDependencies": {
-    "@commitlint/config-conventional": "^17.0.2",
+    "@commitlint/config-conventional": "^19.6.1",
```

The latest available version is `19.6.0`
```suggestion
    "@commitlint/config-conventional": "^19.6.0",
```

---

> The fix has been added to the library I am working on, and the issue has been resolved.
@yjose could you leave a link to the library you are working on or a minimum reproducable example so we can test it?

---

correct, i am using 19.5.0 in my setup but i suppose the 19.6.0 should work too